baseapp/abci.go

```diff
@@ -130,7 +130,7 @@ func (app *BaseApp) SetOption(req abci.RequestSetOption) (res abci.ResponseSetOp
 func (app *BaseApp) Info(req abci.RequestInfo) abci.ResponseInfo {
 	lastCommitID := app.cms.LastCommitID()
 	// load the app version for a non zero height and zero app hash
```

Comment:
[optional] this comment seems confusing b/c it doesn't immediately apply to the conditional below it
```suggestion
```

Reply:
I think it still applies because you are still loading the app version if it hasn't been set